OracleDataIntegrator/README.md

```diff
@@ -24,6 +24,8 @@ You need to have a running database container or a database running on any machi
 The database connection details are required for creating ODI specific RCU schemas while configuring ODI domain. 
 While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
 
+The Oracle Database image can be pulled from the [DockerStore](https://store.docker.com/images/oracle-database-enterprise-edition) or the [Oracle Container Registry](https://container-registry.oracle.com) or you can build your own using the Dockerfiles and scripts in [GitHub Location](https://github.com/oracle/docker-images/tree/master/OracleDatabase/dockerfiles/12.2.0.1)
```

Comment:
It's "Docker Store": there's a space.  I'm being specific about this because it's a trademark of Docker and we need to get this stuff right. :)